index.html

```diff
@@ -230,6 +230,27 @@
         <dt>
           midi
         </dt>
+	<dt>
+	  camera
+	</dt>
+	<dt>
+	  microphone
+	</dt>
+	<dt>
+	  speaker
+	</dt>
+	<dt>
+	  all-cameras
+	</dt>
+	<dt>
+	  all-microphones
+	</dt>
+	<dt>
+	  all-speakers
```

Comment:
Very possible. The descriptor is only visible as a query parameter, so omitting the deviceId should be OK. Wouldn't even change the IDL, since dictionary elements are by default optional.